src/platform/intel/cavs/platform.c

```diff
@@ -245,7 +245,7 @@ static void platform_memory_windows_init(void)
 	/* window2, for debug */
 	io_reg_write(DMWLO(2), HP_SRAM_WIN2_SIZE | 0x7);
 	io_reg_write(DMWBA(2), HP_SRAM_WIN2_BASE
-		| DMWBA_READONLY | DMWBA_ENABLE);
+		| DMWBA_ENABLE);
```

Comment:
We should also add a flag in FW_READY message (alongside the other debug flags) to indicate GDB debug is supported in the build.

Reply:
Yes, this will be covered in next commit where I deal with IPC communication between testing tool <--> SoF.